src/api-extension/utils/checkIfItemsQuantityIsEqualOrHigherThanItemTotalQuantityDiscount.ts

```diff
@@ -0,0 +1,24 @@
+import { LineItem } from '@commercetools/platform-sdk';
+
+export default function checkIfItemsQuantityIsEqualOrHigherThanItemTotalQuantityDiscount(
+  lineItems: LineItem[],
+): boolean {
+  return !!lineItems?.find((lineItem) => {
+    if (lineItem.custom?.fields?.applied_codes) {
+      const { quantity: itemQuantity } = lineItem;
+      const appliedCodes = (lineItem.custom?.fields?.applied_codes ?? [])
```

Comment:
you can probably shorten it to:
`const totalQuantityDiscount = (lineItem.custom?.fields?.applied_codes ?? [])
        .map((code) => JSON.parse(code))
        .filter((code) => code.type === 'UNIT')
        ,filter((code) => code.quantity)
        .reduce((sum, codeObject) => sum += codeObject.quantity, 0);`